src/components/uielements/label/label.style.ts

```diff
@@ -48,6 +49,7 @@ const colors: Colors = {
 
 export const LabelWrapper = styled.div`
   padding: 10px 0;
+  text-align: ${(props: Props) => props.align};
```

Comment:
`props.align` can be `undefined`.
I think it would be great if you can add default value here.

Reply:
The default is defined in the src/components/uielements/label/label.tsx component. This should be good enough right? See https://github.com/thorchain/asgardex-electron/blob/20bb34a44bce22be2b4082e4f4c320f30474e61c/src/components/uielements/label/label.tsx#L20

Reply:
@FrozenIce0617 is right (good catch btw). A default value in `label.tsx` won't save you, as long as `align?: TextAlignment` is defined as an optional value.

Reply:
If the `align` has a default value, we can define `align` as a required props. :)